api/src/models/ImageFile.ts

```diff
@@ -71,21 +71,50 @@ class ImageFile {
     }
 
     ocrDerivative() {
-
+        let fs = require("fs");
+        let png = this.derivativePath('ocr/pngs', 'png');
+        let { exec } = require("child_process");
+        if (fs.existsSync(png)) {
+            let tc_cmd = "#{config['textcleaner_path']} #{config['textcleaner_switches']} #{derivative('LARGE')} #{png}"
+            exec(tc_cmd, (error, stdout, stderr) => {
+                if (error) {
+                    console.log(`error: ${error.message}`);
+                    return;
+                }
+                if (stderr) {
+                    console.log(`stderr: ${stderr}`);
+                    return;
+                }
+                console.log(`stdout: ${stdout}`)
+            });
+        }
     }
 
     ocrProperties() {
-
+        let fs = require("fs");
+        var path = require('path');
+        let file = this.derivativePath('ocr/pngs', 'png');
```

Comment:
I think this is copied and pasted from the ocrDerivative() method, but for this you just want to create 'ocr/tesseract.config' inside the current job directory.